src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -177,6 +191,56 @@ public PrivilegesEvaluator(
         termsAggregationEvaluator = new TermsAggregationEvaluator();
         pitPrivilegesEvaluator = new PitPrivilegesEvaluator();
         this.namedXContentRegistry = namedXContentRegistry;
+
+        if (configurationRepository != null) {
+            configurationRepository.subscribeOnChange(configMap -> {
```

Comment:
what kind of events will this subscribe listen to?

Reply:
Whenever a user uploads a new role or action group configuration.